src/plugins/intel_npu/thirdparty/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(BUILD_SHARED_LIBS OFF)
+
+#
+# zeroApi
+#
+
+if(ENABLE_ZEROAPI_BACKEND)
+    # We have to avoid linking against loader with debug postfix due it's a part of driver
+    # and the name will be the same for release and debug configurations
+    set(OLD_CMAKE_DEBUG_POSTFIX ${CMAKE_DEBUG_POSTFIX})
```

Comment:
we don't need to restore these value, because it affects only `src/plugins/intel_npu/thirdparty/` folder.